api/queue/get.js

```diff
@@ -0,0 +1,41 @@
+const queue = require('./queue.js');
+const bot   = require('../bot.js');
+
+module.exports = (ids, author) => {
+  const promises = [];
+
+  ids.forEach((id) => {
+    promises.push(queue.getJob(id));
+  });
+
+  Promise.all(promises).then((res) => {
+    // console.log(1, res);
+    const jobs = [];
+    for (let i = 0; i < res.length; i++) {
+      if (res[i]) {
+        jobs.push(res[i]);
+      }
+    }
+
+    if (jobs.length > 0) {
+      const job = {};
```

Comment:
youre using a hash here to reduce another hash two keys. this would be fine if the hash needed to persist in memory but you use it immediately and reset the values

if anything, I think it would make more sense to either just use `jobs[j].data` directly in the `reminders` string or (if cleanliness is the issue) change this to `job_data` and set `jobs[j].data` equal to it in the loop.
should look about the same but get rid of 2 lines